CRM/Core/Permission/UnitTests.php

```diff
@@ -43,9 +43,8 @@ public function check($str, $userId = NULL) {
     if ($str == CRM_Core_Permission::ALWAYS_ALLOW_PERMISSION) {
       return TRUE;
     }
-
     // return the stubbed permission (defaulting to true if the array is missing)
-    return is_array($this->permissions) ? in_array($str, $this->permissions) : TRUE;
+    return isset($this->permissions) && is_array($this->permissions) ? in_array($str, $this->permissions) : TRUE;
```

Comment:
My personal pref is to clarify the groupings with brackets and it's not unheard of for php to change the order between versions (in fact in looks like there's a minor change for `.` in 8 - not relevant here), but ok this looks right.